mne/io/brainvision/brainvision.py

```diff
@@ -498,11 +498,16 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale):
     for chan, props in cfg.items('Channel Infos'):
         n = int(re.findall(r'ch(\d+)', chan)[0]) - 1
         props = props.split(',')
+
         # default to microvolts because that's what the older brainvision
         # standard explicitly assumed; the unit is only allowed to be
         # something else if explicitly stated (cf. EEGLAB export below)
         if len(props) < 4:
-            props += (u'µV',)
+            props += ('µV',)
+        elif props[3] == '':
+            # deal with files where the unit property is simply empty
```

Comment:
```suggestion
            # default to µV when unit property is empty, according to BV spec
```

Reply:
I've reworked things a bit in 219b492 to avoid redundancy

Reply:
great, thanks